third_party/ibis/ibis_postgres/registry.py

```diff
@@ -23,6 +23,12 @@ def sa_format_hashbytes(translator, op):
 
 
 def sa_epoch_seconds(translator, op):
-    """Override for standard ExtractEpochSeconds but catering for larger second values."""
+    """Override for standard ExtractEpochSeconds but catering for larger second values.
+
+    This expression also truncates fractional seconds from the incoming datetime.
+    This matches behaviour or other SQL engines' epoch seconds expressions."""
```

Comment:
```suggestion
    This matches behaviour on other SQL engines' epoch seconds expressions."""
```
nitpicking, small typo